src/main/java/micdoodle8/mods/galacticraft/core/entities/EntityBubble.java

```diff
@@ -7,7 +7,7 @@
 //import micdoodle8.mods.galacticraft.core.tile.TileEntityOxygenDistributor;
 //import micdoodle8.mods.galacticraft.core.util.ConfigManagerCore;
 //import micdoodle8.mods.galacticraft.core.util.GCLog;
-//import micdoodle8.mods.miccore.Annotations.NetworkedField;
+//import micdoodle8.mods.galacticraft.core.util.Annotations.NetworkedField;
 //import net.minecraft.entity.player.EntityPlayer;
 //import net.minecraft.nbt.NBTTagCompound;
 //import net.minecraft.tileentity.TileEntity;
```

Comment:
since repository is version controlled - dead code is bad, it rots (in this example IDEA was smart enough to update the reference, but it doesn't always work), I would suggest removing it